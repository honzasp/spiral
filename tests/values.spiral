(program
  (fun three-args (x y z)
    (let ((x+y (+ x y))
          (y+z (+ y z)))
      (println x+y)
      (println y+z)))
  (three-args 3 10 17)
  
  (fun sum-n (limit)
    (do  ((i 0 (+ i 1))
          (sum 0 (+ sum i)))
        ((>= i limit)
          sum)))
  (println (sum-n 10))
)
          
