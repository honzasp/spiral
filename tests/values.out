13
27
55
